CHANGELOG.rst

```diff
@@ -4,6 +4,15 @@ Changelog
 in development
 --------------
 
+Added
+~~~~~
+
+* Added st2 API get action parameters by ref.
```

Comment:
Please add reference to the pull request, e.g. #5509 (see the other formats in the file). That way its easy to track PRs from changelog.
Also each new entry doesn't need a header "added", just add to the existing "Added" section already in place for the current "in development" section.